packages/experiments/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -79,46 +71,72 @@ export const SplitButtonStyles: ISplitButtonComponent['styles'] = (props, theme,
       }
     ],
     button: {
-      borderColor: 'transparent',
+      borderBottomLeftRadius: tokens.borderRadius as string,
+      borderBottomRightRadius: '0px',
+      borderTopLeftRadius: tokens.borderRadius as string,
+      borderTopRightRadius: '0px',
+      borderBottomWidth: tokens.borderWidth,
+      borderLeftWidth: tokens.borderWidth,
+      borderRightWidth: 0,
+      borderTopWidth: tokens.borderWidth,
+      minHeight: tokens.minHeight,
       minWidth: tokens.minWidth,
 
       selectors: {
         '> *': {
           padding: tokens.contentPadding
         },
         ':hover': {
-          borderColor: 'transparent'
+          borderColor: props.primaryActionDisabled ? 'transparent' : tokens.borderColorHovered
         },
         ':active': {
-          borderColor: 'transparent'
+          borderColor: props.primaryActionDisabled ? 'transparent' : tokens.borderColorPressed
         }
       }
     },
     splitDivider: {
-      backgroundColor: tokens.color,
+      backgroundClip: 'content-box',
+      backgroundColor: tokens.dividerColor,
+      borderBottomColor: tokens.borderColor,
+      borderTopColor: tokens.borderColor,
+      borderStyle: 'solid',
+      borderBottomWidth: tokens.borderWidth,
+      borderLeftWidth: 0,
+      borderRightWidth: 0,
+      borderTopWidth: tokens.borderWidth,
       boxSizing: 'border-box',
-      height: 'calc(100% - 14px)',
-      margin: '7px 0px',
+      height: '100%',
+      padding: '7px 0px',
       width: 1,
 
       selectors: {
         [HighContrastSelector]: {
           borderColor: tokens.highContrastColor
+        },
+        ':hover': {
+          borderColor: tokens.borderColorHovered
+        },
+        ':active': {
+          borderColor: tokens.borderColorPressed
         }
       }
     },
     menuButton: {
-      borderColor: 'transparent',
+      borderBottomLeftRadius: '0px',
+      borderBottomRightRadius: tokens.borderRadius as string,
+      borderTopLeftRadius: '0px',
+      borderTopRightRadius: tokens.borderRadius as string,
```

Comment:
What type error do you get without the cast?

Reply:
For some reason `borderRadius` is type as `number | string` but its variants `borderBottomLeftRadius`, `borderBottomRightRadius`, `borderTopLeftRadius` and `borderTopRightRadius` are typed only as `string`.

Reply:
I think it's worth fixing [border corner radii in IRawStyleBase](https://github.com/OfficeDev/office-ui-fabric-react/blob/12772428d745ee70303fc750ff932093dfc4c3bf/packages/merge-styles/src/IRawStyleBase.ts#L661) to be similar to `borderRadius?: ICSSRule | ICSSPixelUnitRule;`